scout/adapter/mongo/case.py

```diff
@@ -587,6 +587,8 @@ def load_case(self, config_data, update=False, keep_actions=True):
 
         old_sanger_variants = self.case_sanger_variants(case_obj["_id"])
 
+        genome_build = str(parsed_case.get("genome_build", 37))
```

Comment:
Seeing these recurrent problems makes me think once again that we should have defined a more rigid structure in the code when it comes to passing genome builds. This code is working if you have either 37 or 38 as genome_build at the case level. But it would probably fail if you have "GRCh37" or "GRCh38" (we have a few of these cases in our database).

What about introducing an additional control to force the build to be in [37,38]? Doesn't have to be here, perhaps even before, like here: https://github.com/Clinical-Genomics/scout/blob/e32ef43598191d2110ffa6e26700c2f87fe75772/scout/parse/case.py#L495

Reply:
I completely agree. It's a bit of a mess throughout the code base. "human_genome_build" is inconsistently treated as a string or an integer (I blame the yaml file format for this (death to yaml!)), and sometimes it defaults to 37 and other times not.

I've added a commit that enforces that "human_genome_build" is either 37 or 38. But this breaks loading cases without the yaml file (ie. with --vcf/--ped), since there is no way to specify the genome_build. Not sure what the best solution to this whole situation is. Any suggestions?

Reply:
Yes, I see your point here. But how does it work now to upload cases from ped file now? (never done it myself). Do these cases get a default genome build 37? Because that would be conceptually wrong anyway

Reply:
No, they get genome build None, see the demo case uploaded using master branch and command: `scout --demo load case --owner cust000 --ped scout/demo/643594.ped --vcf scout/demo/643594.clinical.vcf.gz`
![image](https://user-images.githubusercontent.com/28093618/105696026-119d6380-5f03-11eb-93f5-cdbf1b9247da.png)

Reply:
I don't think having cases without genome build makes a lot of sense. We could solve this issue by adding an extra param here https://github.com/Clinical-Genomics/scout/blob/e32ef43598191d2110ffa6e26700c2f87fe75772/scout/commands/load/case.py#L44

To specify the genome build (this click param could accept only 37 or 38 for instance.
And then in the parser we could have 
```
allowed_genome_builds = ["37", "38"]
    if str(config.get("human_genome_build", **build**)) not in allowed_genome_builds:
        raise ConfigError(f"human_genome_build must be one of {allowed_genome_builds}")
```

Where **build** is passed from the CLI.

Unless we have some reason to allow loading cases without genome build, but I am not aware of any. @dni or @moonso ?

Reply:
But I think for simplicity's sake we could revert to the code of your original PR and merge, and then fix this issue in another one!

Reply:
Sounds like a good idea. I've reverted the last commit. Feel free to merge.

Yes genome_build=None  is the reason the code is littered with various forms of `config.get(build, 37)`. Would be nice to get rid of that mess at some point.

Reply:
I think this is fine to merge now. I've opened a new issue to address the issue we discussed here: #2300